.circleci/config.yml

```diff
@@ -63,26 +63,6 @@ workflows:
     - gradle/test:
           <<: *it-defaults
           test_command: --no-daemon -Dorg.gradle.daemon=false --info it1
-  it2:
-    jobs:
-    - gradle/test:
-          <<: *it-defaults
-          test_command: --no-daemon -Dorg.gradle.daemon=false --info it2
-  it3:
-    jobs:
-    - gradle/test:
-          <<: *it-defaults
-          test_command: --no-daemon -Dorg.gradle.daemon=false --info it3
-  it4:
-    jobs:
-    - gradle/test:
-          <<: *it-defaults
-          test_command: --no-daemon -Dorg.gradle.daemon=false --info it4
-  it5:
-    jobs:
-    - gradle/test:
-          <<: *it-defaults
-          test_command: --no-daemon -Dorg.gradle.daemon=false --info it5
```

Comment:
Once breaking changes with gradle 8 are fixed, we can add back more gradle versions for integration testing.